app/assets/stylesheets/application.scss

```
@@ -0,0 +1,2 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
```

in css and js file, we use `'` instead of `"`
```suggestion
@import 'bootstrap-sprockets';
@import 'bootstrap';
```